src/vaadin-grid-column.html

```diff
@@ -322,6 +322,9 @@
       }
 
       cells.forEach(cell => {
+        if (!cell.parentElement) {
```

Comment:
Let's update the code to only check for `cell.parentElement` once:

```js
const parent = cell.parentElement;
if (!parent) {
  return;
}
const model = this._grid.__getRowModel(parent);
```

Reply:
done.